cacheops/invalidation.py

```diff
@@ -18,16 +18,32 @@
 __all__ = ('invalidate_obj', 'invalidate_model', 'invalidate_all', 'no_invalidation')
 
 
+def delete_invalid_caches(conj_keys):
+    for conj_key in conj_keys:
+        i = 0
+        while True:
+            i, cache_keys = redis_client.sscan(conj_key, cursor=i, count=1000)
+            if cache_keys:
+                redis_client.srem(conj_key, cache_keys)
+                redis_client.delete(cache_keys)
+            if i == 0:
+                break
+    redis_client.delete(conj_keys)
+
+cleanup_fn = getattr(settings, 'CACHEOPS_CLEANUP_FN', delete_invalid_caches)
```

Comment:
I don't want to add RQ as a requirement in cacheops. This way, the application using cacheops can pass in a function as a setting that replaces the default invalidation function above – in Rover's case, it'll be a function that queues an RQ task.